### PR TITLE
Build asset for the legacy module - Closes #50

### LINKED ISSUE #50: Build asset for the Legacy Module
### Description

The migrator tool must iterate through the Lisk Core node’s state store to build the block asset for the Legacy module following the steps specified in LIP-0063.

### Acceptance Criteria

- `addLegacyModuleEntry` is implemented
- Unit tests are written

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #50 

### How was it solved?
- [x] addLegacyModuleEntry is implemented
- [x] Unit tests are written

### How was it tested?
- Local by running unit tests

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
I think the balance logic is missing

File: test/unit/assets/legacy.spec.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			legacyModuleAsset = new LegacyModuleAsset(db);
+		});
+
+		it('should get legacy accounts', async () => {
+			const response = await legacyModuleAsset.addLegacyModuleEntry();
+
+			// Assert
+			expect(response.module).toEqual(MODULE_NAME_LEGACY);
+			expect(response.data).toHaveProperty('accounts');
+			expect(response.data.accounts).toBeInstanceOf(Array);
```
Comment (ManuGowda) [inline test/unit/assets/legacy.spec.ts:36]: maybe here you can if the accounts has address and balances? array instance check only ensures its an array not the values of accounts?

expect response.data.accounts.length check to have more than 0
expect(Object.keys(response.data.accounts[0]).toEqual(['address', 'balance']))

**Review by ManuGowda — COMMENTED** [review]
I am not sure if we need the fixture? `test/unit/assets/fixtures/blockchain.db/OPTIONS-000100`. If its only for the test, we can mock the `KVStore` to return dummy response?

for not this is not big concern i see the fixture getting increased if we have all modules.

File: test/unit/assets/legacy.spec.ts (hunk 1 above)
Comment (nagdahimanshu) [reply test/unit/assets/legacy.spec.ts]: I mocked the response and updated the test accordingly.

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**